cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -238,8 +250,6 @@ def load_resource_file(
                 "authentication",
                 "sourceOidcCredentials",
                 "destinationOidcCredentials",
-                "sourceNonce",
-                "destinationNonce",
```

Comment:
getting a bit lost here, what's the purpose of `auth_dict`? It seems to diverge from code around: sourceOidcCredentials is removed from item for example but not from auth_dict